tbx/static_src/sass/config/_mixins.scss

```diff
@@ -201,6 +201,16 @@
     }
 }
 
+/* ============================================
+    High Contrast mode in light mode
+    prefers-color-scheme: light + forced-colors: active
+*/
+@mixin high-contrast-light-mode() {
+    @media (prefers-color-scheme: light) and (forced-colors: active) {
+        @content;
+    }
+}
```

Comment:
I think the high contrast mixin [we have here](https://github.com/torchbox/torchbox.com/blob/integration/sprint-2/tbx/static_src/sass/config/_mixins.scss#L208) would be the same as the above as the browser defaults to the light color scheme preference unless dark is selected but perhaps it doesn't do any harm to be explicit.

Reply:
I didn't want to override it for the dark one, which I'd have had to do if I changed the default - so I think this is ok.